src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/HearingDuration.java

```diff
@@ -1,733 +1,731 @@
 package uk.gov.hmcts.reform.sscs.ccd.domain;
 
-import static java.util.Objects.isNull;
 import static java.util.Objects.nonNull;
 import static org.apache.commons.collections4.CollectionUtils.isNotEmpty;
-import static org.apache.commons.lang3.ObjectUtils.isEmpty;
 import static uk.gov.hmcts.reform.sscs.ccd.domain.BenefitCode.*;
 import static uk.gov.hmcts.reform.sscs.ccd.domain.Issue.*;
 
 import java.util.*;
-import java.util.stream.Collectors;
 import lombok.AllArgsConstructor;
 import lombok.Getter;
+import org.elasticsearch.core.Tuple;
 
 @AllArgsConstructor
 @Getter
 public enum HearingDuration {

```

Comment:
@dattipoeHMCTS just imagine you had to update this enum.. I would like to see your reaction to seeing this for the first time, we should not be doing this.